md/codeBlock/eval.ts

```diff
@@ -28,10 +29,8 @@ export type EvaluateOptions = {
   replace?: [string | RegExp, string][];
 };
 
-type Details = ReturnType<typeof parseCodeBlock>;
```

Comment:
This is one of the clumsiest parts in the existing code. Glad to see it resolved.